Sources/RichEditorView/Resources/editor/rich_editor.js

```diff
@@ -24,6 +24,60 @@ document.addEventListener("selectionchange", function() {
     RE.backuprange();
 });
 
+document.addEventListener('click', function(event) {
+  // This code will be executed whenever a click occurs anywhere inside the document.
+  // You can replace this code with any action you want to perform when a click happens.
+     var items = [];
+
+     if (document.queryCommandState('bold')) {
```

Comment:
this is very repetitive. Query command state is also deprecated and may not work if you load in HTML with certain formats already applied